tests/src/system/basic/WskActionTests.scala

```diff
@@ -0,0 +1,268 @@
+ * limitations under the License.
+ */
+
+package system.basic
+
+import org.junit.runner.RunWith
+import org.scalatest.junit.JUnitRunner
+
+import common.JsHelpers
+import common.TestHelpers
+import common.TestUtils
+import common.Wsk
+import common.WskProps
+import common.WskTestHelpers
+import spray.json._
+import spray.json.DefaultJsonProtocol._
+import spray.json.JsObject
+import spray.json.pimpAny
+
+@RunWith(classOf[JUnitRunner])
+class WskActionTests
+    extends TestHelpers
+    with WskTestHelpers
+    with JsHelpers {
+
+    implicit val wskprops = WskProps()
+    val wsk = new Wsk(usePythonCLI = false)
+
+    val testString = "this is a test"
+    val testResult = JsObject("count" -> testString.split(" ").length.toJson)
+
+    behavior of "Whisk actions"
+
+    it should "invoke an action with a space in the name" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "hello Async"
+            assetHelper.withCleaner(wsk.action, name) {
+                (action, _) => action.create(name, Some(TestUtils.getCatalogFilename("samples/helloAsync.js")))
+            }
+
+            val run = wsk.action.invoke(name, Map("payload" -> testString.toJson))
+            withActivation(wsk.activation, run) {
+                activation =>
+                    activation.response.status shouldBe "success"
+                    activation.response.result shouldBe Some(testResult)
+                    activation.logs.get.mkString(" ") should include(testString)
+            }
+    }
+
+    it should "pass parameters bound on creation-time to the action" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "printParams"
+            val params = Map(
+                "param1" -> "test1",
+                "param2" -> "test2")
+
+            assetHelper.withCleaner(wsk.action, name) {
+                (action, _) =>
+                    action.create(
+                        name,
+                        Some(TestUtils.getCatalogFilename("samples/printParams.js")),
+                        parameters = params.mapValues(_.toJson))
+            }
+
+            val invokeParams = Map("payload" -> testString)
+            val run = wsk.action.invoke(name, invokeParams.mapValues(_.toJson))
+            withActivation(wsk.activation, run) {
+                activation =>
+                    val logs = activation.logs.get.mkString(" ")
+
+                    (params ++ invokeParams).foreach {
+                        case (key, value) =>
+                            logs should include(s"params.$key: $value")
+                    }
+            }
+    }
+
+    it should "copy an action and invoke it successfully" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val name = "copied"
+            assetHelper.withCleaner(wsk.action, name) {
+                (action, _) => action.create(name, Some("/whisk.system/samples/wordCount"), Some("copy"))
```

Comment:
After the catalog is removed, the namespace whisk.system does not exist and the package samples does not exist as well.
You may need to create the package samples and the action wordCount before creating the copy of the action.